recipes/libharu/all/conanfile.py

```diff
@@ -39,16 +42,19 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
+            del self.options.with_libtiff
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
 
     def requirements(self):
         self.requires("zlib/1.2.11")
         self.requires("libpng/1.6.37")
+        if self.options.get_safe("with_libtiff"):
+            self.requires("libtiff/4.2.0")
```

Comment:
There is another solution: rename conflicting symbols (at least `TIFFFaxWhiteCodes`) in libharu source code.

Reply:
done